docker-compose.yml

```diff
@@ -32,6 +32,8 @@ services:
       - DB_PASSWORD=mypassword
       - DB_DATABASE=testdb
       - ASPNETCORE_ENVIRONMENT=Test
+      - ADDRESSES_API_BASE_URL=Test
+      - ADDRESSES_API_KEY=Test
```

Comment:
I understand this is added locally to get the container building so that tests would pass. However, this won't make the integration tests pass for the endpoints that rely on the call to the API.

I think we're better off removing it, and spinning up the docker container locally with the "docker compose run" and passing in "-e key=val" flags like I did it in the circleci config.

Reply:
Because we are using docker-compose up in the makefile we will probably need to change this around as well in order to pass in the env variables.  Probably a bit much for this PR.